### PR TITLE
Database Repository using Doctrine ORM

### PR DESCRIPTION
**Pull request by CarolinWickner**
- Made database running using Doctrine ORM

### CODE REVIEW

File: src/AppBundle/Repository/DatabaseProjectRepository.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+declare (strict_types = 1);
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Project;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+
+class DatabaseProjectRepository extends EntityRepository implements ProjectRepository
```
Comment (mfranczak) [inline src/AppBundle/Repository/DatabaseProjectRepository.php]: Nice wrapper :)

File: composer.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "require": {
         "php": ">=5.5.9",
         "doctrine/doctrine-bundle": "^1.6",
+        "doctrine/doctrine-migrations-bundle": "~1.2",
```
Comment (mfranczak) [inline composer.json]: Migration to create database table is missing  in the PR.

File: composer.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
     "require": {
```
Comment (mfranczak) [inline composer.json]: It's a bit weird that you added a new dependency but composer.lock is not in the PR.

File: composer.json [hunk 4]
```diff
@@ -20,26 +20,28 @@
         ]
     },
     "require": {
-        "php": ">=5.5.9",
-        "doctrine/doctrine-bundle": "^1.6",
+        "php": "^7.1",
```
Comment (mfranczak) [inline composer.json]: :)

File: composer.json [hunk 5]
```diff
@@ -20,26 +20,28 @@
...
-        "symfony/validator": "*"
+        "symfony/validator": "*",
+        "twig/twig": "^1.0||^2.0"
     },
     "require-dev": {
+        "phpunit/phpunit": "^6.5",
         "sensio/generator-bundle": "^3.0",
-        "symfony/phpunit-bridge": "^3.0",
-        "phpunit/phpunit": "^6.5"
+        "symfony/maker-bundle": "^1.11",
```
Comment (mfranczak) [inline composer.json]: what is this bundle for?

Comment (CarolinWickner) [reply composer.json]: I had some issues with composer and did a lot of stuff to make it work. Turns out I don't need this.

**Pull request merged**